src/cpc_sketch_pg_functions.c

```diff
@@ -286,7 +368,6 @@ Datum pg_cpc_sketch_union(PG_FUNCTION_ARGS) {
     cpc_sketch_delete(sketchptr2);
   }
   sketchptr = cpc_union_get_result(unionptr);
-  cpc_union_delete(unionptr);
```

Comment:
Unclear why this was removed? It'll be cleaned up at the end of the transaction, but it seems to have been a temporary local variable?

Reply:
now get_result deletes the union